odl/discr/lp_discr.py

```diff
@@ -785,6 +997,60 @@ def uniform_discr(min_corner, max_corner, nsamples,
                                    **kwargs)
 
 
+def sequence_space(shape, exponent=2.0, impl='numpy', **kwargs):
+    """Return an object mimicing the sequence space ``l^p(R^d)``.
```

Comment:
Right. In practice, the only difference is the multi-dimensionality of the indices.